source/documentation/rdfs/__index.md

```diff
@@ -137,7 +139,7 @@ curl --data @query.rq \
 
 will return:
 
-```
+```sparql
```

Comment:
sparql syntax doesn't apply to the test format results.

Reply:
I removed the sparql syntax bit :+1: